internal/invocation/stripe_executor_test.go

```diff
@@ -0,0 +1,139 @@
+package invocation
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/goleak"
+)
+
+type orderChecker struct {
+	*testing.T
+	previousCallArg int
```

Comment:
`previousCallArg` should be protected with atomic, wherever it is accessed, since it is written after initialization and accessed by several goroutines.

Reply:
Of course! My bad, fixed